webtransport/quic/client-indication.any.js

```diff
@@ -0,0 +1,32 @@
+// META: quic=true
+// META: script=/common/get-host-info.sub.js
+
+const PORT = 8983;
+const HOST = get_host_info().ORIGINAL_HOST;
```

Comment:
This may or may not be better written as:
```js
const { ORIGINAL_HOST: HOST, ORIGIN } = get_host_info();
```

Reply:
Done.